### PR TITLE
Added styling on Patients page Issue#2735

### LINKED ISSUE #2735: Styling is missing on Patients page
https://care.coronasafe.in/facility/551f9e09-04a3-4f52-ade2-1db45588741d/patient/6916690c-66f5-48d4-a723-2365ac5da1cf

Styling for  "**Consultation History**" and "**Sample Test History**" is absent
Screenshot (389)

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #2735 
Added styling for "**Consultation History**" and "**Sample Test History**" 
Screenshot (388)

### CODE REVIEW

**Comment by rithviknishad** [comment]
netlify build failing even after clear cache.
vercel deploy seems to work though.
@gigincg @nihal467

**Review by developedBySJ — APPROVED** [review]
LGTM

**Review by cp-Coder — APPROVED** [review]

**Comment by gigincg** [comment]
@Pranshu1902 Can you merge the latest develop?

**Comment by Pranshu1902** [comment]
@gigincg done

**Comment by nihal467** [comment]
@Pranshu1902 


test failed

**Comment by Pranshu1902** [comment]
Updated view:
image

**Comment by nihal467** [comment]
@Pranshu1902 test failed

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

**Comment by nihal467** [comment]
@Pranshu1902 
image

- [ ] when the sent to collection center, button comes up, on the left side, the right side card is using a lot of extra space, to counter balance it, so better change the sent to collection centre button to right  side, so the card stays in a better shape
- [ ] the created and modify details look so messed up, come up with something user friendly

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

Mobile View:
image

**Comment by nihal467** [comment]
@Pranshu1902 

image

the button placing doesn't look good, can we have some better positioning

**Comment by Pranshu1902** [comment]
@nihal467 
how's this?
image

If I place it in a separate line then there will be issue with spacing in neighbouring card

**Comment by nihal467** [comment]
@Pranshu1902 still doesn't look good, can you take an opinion from others by posting it in slack

**Comment by Pranshu1902** [comment]
@nihal467 
Updated view:
image

**Comment by nihal467** [comment]
test approved

File: src/Components/Patient/SampleTestCard.tsx [hunk 1]
```diff
@@ -89,54 +89,41 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
             <span className="text-gray-700">Result: </span>
             {_.startCase(_.camelCase(itemData.result))}
           </div>
-          <div className="md:col-span-2">
+          <div className="">
```
Comment (khavinshankar) [inline src/Components/Patient/SampleTestCard.tsx]:
```suggestion
          <div>
```

File: src/Components/Patient/SampleTestCard.tsx [hunk 2]
```diff
@@ -89,54 +89,41 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
...
             <span className="text-gray-700">Sample Type: </span>
             {itemData.sample_type !== "OTHER TYPE"
               ? itemData.sample_type
               : itemData.sample_type_other}
           </div>
           {itemData.fast_track && (
-            <div className="md:col-span-2">
+            <div className="">
```
Comment (khavinshankar) [inline src/Components/Patient/SampleTestCard.tsx]:
```suggestion
            <div>
```

File: src/Components/Patient/SampleTestCard.tsx [hunk 3]
```diff
@@ -89,54 +89,41 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
...
 
-        <div className="mt-4">
-          <div className="text-gray-600 text-sm font-bold">
-            <span className="text-gray-800">Date of Sample:</span>{" "}
-            {itemData.date_of_sample
-              ? moment(itemData.date_of_sample).format("lll")
-              : "Not Available"}
-          </div>
+        <div className="mt-4 flex flex-col md:flex-row justify-between gap-4">
+          <div className="">
```
Comment (khavinshankar) [inline src/Components/Patient/SampleTestCard.tsx]:
```suggestion
          <div>
```

**Review by khavinshankar — COMMENTED** [review]
just some minor changes, remove all the empty classNames

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**